lib/db-util.js

```diff
@@ -2,6 +2,7 @@ module.exports.getDBHostURI = function () {
     switch (process.env.NODE_ENV) {
         case "production":
         case "development":
+        case "test":
```

Comment:
I may not need this anymore since E2E tests use production env settings now, but may still be worth having this around so that the MongoDB URI can be changed for integ tests via config.